segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -580,6 +620,15 @@ public void mergeSegmentsBatch(WireCommands.MergeSegmentsBatch mergeSegments) {
                });
     }
 
+    private CompletableFuture<Void> appendOnIndexSegment(String segmentName) {
+            return segmentStore.getAttributes(getIndexSegmentName(segmentName), Collections.singleton(EXPECTED_INDEX_SEG_EVENT_SIZE), true, TIMEOUT)
+                    .thenApply(attributes -> attributes.getOrDefault(EXPECTED_INDEX_SEG_EVENT_SIZE, 0L))
+                    .exceptionally(e -> {
+                        log.warn("Exception occured while getting max event size for index segment {}, exception: {}", getIndexSegmentName(segmentName), e);
+                        return 0L;
```

Comment:
Instead of default value as 0, it can be NameUtils.INDEX_APPEND_EVENT_SIZE .

Reply:
See above